torch/install-torch.sh

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
+#
+# demo job to test the pbig partition
+#   job name is big hello (yeah, big job,small result as usual)
+# 
+#SBATCH --job-name="torch-install"
+#SBATCH --output=torch-install.out
+#SBATCH --error=torch-install.out
+#
+# run on all cores minus one of the node, require 2GB per core = 14GB
+#SBATCH --nodes=1
+#SBATCH --ntasks-per-node=7
+#SBATCH --mem-per-cpu=2048
```

Comment:
You may want to request a GPU here, just in case. I know the sruns which need it, request it, but I don't know what happens in a batch context if your batch job ends up allocated on a non-GPU node (though we may not even have those on Euryale!).